tests/plugins/link/manual/anchorcustomstyles.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.16.1, bug, link, 4728
+@bender-ui: collapsed
+@bender-ckeditor-plugins: link, toolbar, wysiwygarea, basicstyles, sourcearea, clipboard, enterkey, link, list, liststyle, tabletools, tableselection, undo, format
```

Comment:
```suggestion
@bender-ckeditor-plugins: link, toolbar, wysiwygarea, basicstyles, sourcearea, clipboard, enterkey, link, list, liststyle, tabletools, tableselection, undo, format, elementspath
```

I would add `elementspath` to plugins. It shows what kind of element is in current caret position, which is important from the perspective of this issue.